notes

```diff
@@ -0,0 +1,2 @@
+install vendor url:
```

Comment:
Might want to use a more descriptive filename/add an extension (like `TODO.md` or `NOTES.md`), just to make it clear what this file is just from the name

Reply:
done https://github.com/flamingchickens1540/bunnybots-elmo-2022/blob/micah/USEFUL_NOTES.md